src/App.js

```diff
@@ -53,12 +64,27 @@ class App extends Component {
     }
   }
 
+  fetchRegistrationManagement = async () => {
+    try {
+      const response = await registrationManagementService.get()
+      this.props.setRegistrationManagement(response.registrationManagement)
+    } catch (e) {
+      console.log('error happened', e)
+      this.props.setError(
+        'Error fetching registration management configuration'
+      )
```

Comment:
If this fails, should the registration be closed or open? Perhaps we should set the reducer's initial state to have registration closed by default with some fallback message.

![image](https://user-images.githubusercontent.com/21111572/51789095-3339a800-218e-11e9-973c-1aa4fc87de6c.png)